fl4health/clients/basic_client.py

```diff
@@ -386,8 +386,8 @@ def _log_header_str(
                 training. Defaults to None.
         """
 
-        log_str = f"Current FL Round: {str(current_round)}\t" if current_round is not None else ""
-        log_str += f"Current Epoch: {str(current_epoch)}" if current_epoch is not None else ""
+        log_str = f"Current FL Round: {int(current_round)} " if current_round is not None else ""
```

Comment:
Just curious if there was any reason for the cast to int? Is there situations where this isn't an int or None?

Reply:
I think mypy was complaining at one point about current round being an optional int. Probably was due to something else and I just never changed it back